api/turing/cluster/controller.go

```diff
@@ -218,6 +218,7 @@ func (c *controller) ApplyConfigMap(namespace string, configMap *ConfigMap) erro
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      configMap.Name,
 			Namespace: namespace,
+			Labels:    configMap.Labels,
```

Comment:
I think there are more instances in this file (and perhaps across the codebase) where `ObjectMeta` from `k8s.io/apimachinery/pkg/apis/meta/v1` is used, where we are currently not setting the labels. Might be easy for search for this type and figure out what else needs to be updated, since it seems like all resources rely on the same underlying struct to store the basic info.

Reply:
Thanks for catching this, took a closer look and noticed I missed out `Secrets`, as well as `ServiceAccount`, `Role`, `RoleBinding` which are currently used by Batch Ensembling. I refactored the latter resources to follow the builder pattern instead of adding a new `labels` parameter to their respective create function calls.